libs/navigation/src/drivers/magento/transformers/navigation-transformer.spec.ts

```diff
@@ -22,26 +23,40 @@ describe('Driver | Magento | Navigation | Transformers | DaffMagentoNavigationTr
   });
 
   it('should transform a CategoryNode into a DaffNavigationTree', () => {
-    const navigation = navigationTreeFactory.create();
-
     const categoryNode: CategoryNode = {
-      id: navigation.id,
-      name: navigation.name,
+      id: "1",
+      name: "Root Category",
+      include_in_menu: true,
       products: {
-        total_count: navigation.total_products
+        total_count: 10
       },
-      children_count: navigation.children_count,
+      children_count: 1,
       children: [{
-        id: navigation.children[0].id,
-        name: navigation.children[0].name,
+        id: "2",
+        include_in_menu: true,
+        name: "Subcategory",
         products: {
-          total_count: navigation.children[0].total_products
+          total_count: 10
         },
-        children_count: navigation.children[0].children_count,
+        children_count: 0,
         children: []
       }]
     }
 
+    const navigation : DaffNavigationTreeUnion  = {
+      id: "1",
+      name: "Root Category",
+      total_products: 10,
+      children: [{
+        id: "2",
+        name: "Subcategory",
+        total_products: 10,
+        children: [],
+        children_count: 0,
+      }],
+      children_count: 1,
+    }
```

Comment:
There should be a test case for when the `included_in_menu` is false/null

Reply:
Resolved by b29e6c9197ff8e39dd512711d24a503eb3f19f20